bothan-api/server/src/config/store.rs

```diff
@@ -0,0 +1,31 @@
+use std::path::PathBuf;
+
+use serde::{Deserialize, Serialize};
+
+/// The configuration for all bothan-api's manager.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct StoreConfig {
+    #[serde(default = "default_path", deserialize_with = "deserialize_path")]
```

Comment:
is `deserialize_with` needed? i don't see it anywhere else that you use `PathBuf`?